docs/workflows.rst

```diff
@@ -127,6 +127,9 @@ from additional member(s) before it is accepted as a translation.
 Dedicated reviewers
 -------------------
 
+
+.. versionadded:: 2.18
+
```

Comment:
```suggestion
```

I think it's time to remove these for 2.x versions.

Reply:
Could it link to the relevant changelog instead?
The use-case is making sure things are supported.
Maybe for new changelogs it could go both ways?